package.json

```diff
@@ -14,6 +14,7 @@
     "client-oauth2": "^4.2.1",
     "client-sessions": "^0.8.0",
     "cpr": "^3.0.1",
+    "csurf": "^1.9.0",
```

Comment:
NB: This is kind of unmaintained (last published 3 years ago; older open PRs). https://npms.io/search?q=csurf

I think it's solid though and don't know of any alternatives of the top of my head.